content/en/docs/components/pipelines/installation/tfx-compatibility-matrix.md

```diff
@@ -1,7 +1,7 @@
 +++
-title = "Compatibility Matrix"
+title = "TFX Compatibility Matrix"
```

Comment:
nit: this change is probably unnecessary, we wanted to add more compatibility matrix here.

Reply:
@Bobgy I think it's useful to have a separate page for "TFX Compatalbity Matrix", as the users who actually need to read that page (old users of TFX), are unlikely to click on a page called "Compatibility Matrix"

Also, given that the page is only about TFX right now, leaving it as "Compatibility Matrix" is confusing, and less descriptive.

Finally, "TFX Compatalbity Matrix" should help with search engine optimization.

---

Obviously, you make the final call, these are just my opinions.

Reply:
cc @zijianjoy @kramachandran, what's your opinion on this?
I'm not quite convinced, because again Google search for "kubeflow tfx compatibility" already goes to this page.

I'd prefer not batching this change in this large PR if the topic is controversial, but rather have a separate issue/PR for discussion.

Reply:
@Bobgy it's your guys' call in the end, I am just trying to make the website more "browsable", search engine optimization isn't really the goal for this change.

A page called "Compatibility Matrix" is quite ambiguous to me, it raises the question of what compatibility are we talking about.

Reply:
I think it makes sense to make this more specific. By making this page TFX Compatibility Matrix, we are tying the page to a specific user persona - TFX users.  When a TFX user comes to the KFP website, they probably have some particular/distinct questions form other user types and making the content that they need immediately obvious will be appreciated.  Further, by separating out the TFX compatibility issues from other compatibility issues, we don't make non-TFX users deal with the TFX questions.

Reply:
First, I agree that the discussion of this page's name is outside of Pipelines' doc structure refactoring. Having a separate PR for it can help merging the current PR faster.

Second, have we considered name alternatives for this page, like 'Compatibility Support', 'Dependency Support', 'ML framework compatibility', etc.? There are a few more dependencies which KFP is liable to, which is beyond TFX. I foresee having all of those clarification in one page, instead of multiple pages.

Reply:
@zijianjoy @kramachandran @Bobgy in the interest of getting this merged, I have removed that rename from this PR.

However, I still think we should discuss renaming that page in the future.